src/Umbraco.Web/Security/BackOfficeUserStore.cs

```diff
@@ -454,36 +528,49 @@ public Task<IList<string>> GetRolesAsync(BackOfficeIdentityUser user)
         /// <summary>
         /// Returns true if a user is in the role
         /// </summary>
-        /// <param name="user"/><param name="roleName"/>
+        /// <param name="user"/>
+        /// <param name="normalizedRoleName"/>
+        /// <param name="cancellationToken"></param>
         /// <returns/>
-        public Task<bool> IsInRoleAsync(BackOfficeIdentityUser user, string roleName)
+        public Task<bool> IsInRoleAsync(BackOfficeIdentityUser user, string normalizedRoleName, CancellationToken cancellationToken = default(CancellationToken))
         {
+            cancellationToken.ThrowIfCancellationRequested();
             ThrowIfDisposed();
             if (user == null) throw new ArgumentNullException(nameof(user));
-            return Task.FromResult(user.Roles.Select(x => x.RoleId).InvariantContains(roleName));
+            return Task.FromResult(user.Roles.Select(x => x.RoleId).InvariantContains(normalizedRoleName));
+        }
+
+        public Task<IList<BackOfficeIdentityUser>> GetUsersInRoleAsync(string normalizedRoleName, CancellationToken cancellationToken)
+        {
+            throw new NotImplementedException();
```

Comment:
We could implement this, but can see it's a new thing so can leave for another task. @bergmania  could be an up for grabs task?